src/main/java/fi/otavanopisto/kuntaapi/server/rest/WebPageServiceChannelsApiImpl.java

```diff
@@ -10,14 +10,20 @@
 import org.apache.commons.lang3.EnumUtils;
 
 import fi.metatavu.kuntaapi.server.rest.WebPageServiceChannelsApi;
+import fi.metatavu.kuntaapi.server.rest.model.ElectronicServiceChannel;
 import fi.metatavu.kuntaapi.server.rest.model.WebPageServiceChannel;
+import fi.otavanopisto.kuntaapi.server.controllers.ClientContainer;
 import fi.otavanopisto.kuntaapi.server.controllers.HttpCacheController;
+import fi.otavanopisto.kuntaapi.server.controllers.SecurityController;
 import fi.otavanopisto.kuntaapi.server.controllers.ServiceController;
+import fi.otavanopisto.kuntaapi.server.id.ElectronicServiceChannelId;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'fi.otavanopisto.kuntaapi.server.id.ElectronicServiceChannelId'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)